apps/src/templates/SafeMarkdown.jsx

```diff
@@ -18,6 +18,41 @@ import defaultSanitizationSchema from 'hast-util-sanitize/lib/github.json';
 
 import externalLinks from './plugins/externalLinks';
 
+/**
+ * Basic component for rendering a markdown string as HTML, with sanitization.
+ * Can safely render markdown even from untrusted sources, without potentially
+ * exposing us to an XSS injection.
+ */
+class SafeMarkdown extends React.Component {
```

Comment:
This was moved to the top of the file, the only functional change was adding a className prop so that a class can be passed in to style the block.